bigtable/admin.go

```diff
@@ -1167,14 +1254,21 @@ func (iac *InstanceAdminClient) Clusters(ctx context.Context, instanceID string)
 		if c.EncryptionConfig != nil {
 			kmsKeyName = c.EncryptionConfig.KmsKeyName
 		}
-		cis = append(cis, &ClusterInfo{
+		ci := &ClusterInfo{
 			Name:        nameParts[len(nameParts)-1],
 			Zone:        locParts[len(locParts)-1],
 			ServeNodes:  int(c.ServeNodes),
 			State:       c.State.String(),
 			StorageType: storageTypeFromProto(c.DefaultStorageType),
 			KMSKeyName:  kmsKeyName,
-		})
+		}
+		// Use type assertion to handle protobuf oneof type
+		if cfg, ok := c.Config.(*btapb.Cluster_ClusterConfig_); ok {
```

Comment:
Can you use this instead? https://github.com/googleapis/go-genproto/blob/d629cc9a93d58a60b9f166bf2a825dd80df3879d/googleapis/bigtable/admin/v2/instance.pb.go#L547

Reply:
Yes! Nice find.